aspnet/web-forms/overview/getting-started/getting-started-with-aspnet-45-web-forms/display_data_items_and_details.md

```diff
@@ -15,128 +15,146 @@ by [Erik Reitan](https://github.com/Erikre)
 
 [Download Wingtip Toys Sample Project (C#)](http://go.microsoft.com/fwlink/?LinkID=389434&clcid=0x409) or [Download E-book (PDF)](http://download.microsoft.com/download/0/F/B/0FBFAA46-2BFD-478F-8E56-7BF3C672DF9D/Getting%20Started%20with%20ASP.NET%204.5%20Web%20Forms%20and%20Visual%20Studio%202013.pdf)
 
-> This tutorial series will teach you the basics of building an ASP.NET Web Forms application using ASP.NET 4.5 and Microsoft Visual Studio Express 2013 for Web. A Visual Studio 2013 [project with C# source code](https://go.microsoft.com/fwlink/?LinkID=389434&clcid=0x409) is available to accompany this tutorial series.
+> This tutorial series will teach you the basics of building an ASP.NET Web Forms application using ASP.NET 4.7 and Microsoft Visual Studio Community 2017 for Web. A Visual Studio [project with C# source code](https://go.microsoft.com/fwlink/?LinkID=389434&clcid=0x409) is available to accompany this tutorial series.
 
+In this tutorial, you'll learn how to display data items and data item details using ASP.NET Web Forms and Entity Framework Code First. This tutorial builds on the previous "UI and Navigation" tutorial as part of the Wingtip Toy Store tutorial series. After completing this tutorial, you'll see products on the *ProductsList.aspx* page and a product's details on the *ProductDetails.aspx* page.
 
-This tutorial describes how to display data items and data item details using ASP.NET Web Forms and Entity Framework Code First. This tutorial builds on the previous tutorial "UI and Navigation" and is part of the Wingtip Toy Store tutorial series. When you've completed this tutorial, you'll be able to see products on the *ProductsList.aspx* page and details about an individual product on the *ProductDetails.aspx* page.
+## You'll learn how to:
 
-## What you'll learn:
+- Add a data control to display products from the database
+- Connect a data control to the selected data
+- Add a data control to display product details from the database
+- Retrieve a value from the query string and use that value to limit the data that's retrieved from the database
 
-- How to add a data control to display products from the database.
-- How to connect a data control to the selected data.
-- How to add a data control to display product details from the database.
-- How to retrieve a value from the query string and use that value to limit the data that's retrieved from the database.
-
-### These are the features introduced in the tutorial:
+### Features introduced in this tutorial:
 
 - Model Binding
 - Value providers
 
-## Adding a Data Control to Display Products
+## Adding a Data Control to display products
```

Comment:
Try to shorten H2s where feasible.  This could be "Add a data control" (note lowercase data control)